### ISSUE TITLE
anti-pattern in attribute directives developer guide

### ISSUE THREAD

**Issue by zoechi** [angular/angular.io#1670]
```
@Directive({ selector: '[myHighlight]' })
export class HighlightDirective {
    constructor(el: ElementRef) {
       el.nativeElement.style.backgroundColor = 'yellow';
    }
}
```

Why isn't `@HostBinding('style.backgroundColor) color = 'yellow';` used for this example or not explained that it could be used instead?

**Comment by zoechi** [comment]
If `el.nativeElement` has to be used it should be used with `Renderer` instead of directly accessing the `style` property.

**Comment by thso** [comment]
AFAIK there is no reason, I'll happily merge a PR if you put one together.

**Comment by zoechi** [comment]
Fair enough. I'll have a look.

**Comment by MarkPieszak** [comment]
Agreed, getting people away from touching elements directly, and using Renderer will help people make applications more Universal-like as well.

**Comment by chalin** [comment]
@zoechi: if you don't have a fix in the works, I'll be cleaning this up long the lines of #2164 once it lands. Let me know.

cc @kwalrath

**Comment by Foxandxss** [comment]
There is a pr for this already

**Comment by chalin** [comment]
@Foxandxss: there is the TS side PR, but this morning you notified me that there would be Dart-side work to be done. Are you saying there is already a PR for Dart? If so, I'm not finding it.

**Comment by zoechi** [comment]
Please go ahead. I haven't started yet.

**Comment by wardbell** [comment]
Closed by pr #2164

**Comment by chalin** [comment]
This issue was moved to dart-lang/site-webdev#262
